programs/lfx-mentorship/2025/02-Jun-Aug/project_ideas.md

```diff
@@ -22,7 +22,7 @@
 
 #### Graduate the kubeadm feature gate WaitForAllControlPlaneComponents to GA
 
-- Description:  
+- Description:
```

Comment:
White space is important in markdown (two spaces at the end of a line is a line break)
```suggestion
- Description:  
```